include/openenclave/bits/module.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef _OE_BITS_MODULE_H
+#define _OE_BITS_MODULE_H
+
+/*
+**==============================================================================
+**
+** This file defines functions for loading internal modules that are part of
+** the Open Enclave core.
+**
+**==============================================================================
+*/
+
+#include <openenclave/bits/defs.h>
+#include <openenclave/bits/result.h>
+#include <openenclave/bits/types.h>
+
+OE_EXTERNC_BEGIN
+
+/* Load the host file system module. */
+oe_result_t oe_load_module_hostfs(void);
+
+/* Load the host socket module. */
+oe_result_t oe_load_module_hostsock(void);
```

Comment:
don't use abbreviations ("sock") except to match an ANSI or POSIX standard

Reply:
Fixed.